### DISCUSSION TITLE
Github education application denied multiple times

### DISCUSSION THREAD

**Discussion by GuilleJI** [community/community#127023] [GitHub Education]
### Hi everyone,

As a current student at a college, we were introduced to apply to Github education. Did all the steps to proceed with the application process, and uploaded my student ID, but according to Github, it isnt enough proof to confirm that I'm a student...redid my application more than once, and same response. Do You guys want me to send you a dna sample, 30 testimonials, and cctv tapes to prove once and for all that I go to school? What a joke

**Comment by math-silva** [comment]
In my case, 

I only had to submit a photo of my college ID, and it was accepted. However, some of my classmates were able to get approved by just sending their enrollment verification. Usually, the verification process was completed within a few hours. If you're still having trouble, check with your classmates to see if any of them were successful and ask what they submitted. 

I hope you get your application approved soon!

**Reply by math-silva** [reply]
Make sure you have your institutional email linked to your GitHub account.

**Reply by GuilleJI** [reply]
Thanks for the response. Yeah, I included my institutional email to my Github, and I also submitted other documents such as enrolment verification. Will reach out to one of my classmates and ask him how he got approved.
